pyproject.toml

```diff
@@ -34,6 +32,7 @@ alembic = "^1.4.2"
 python-jose = {extras = ["cryptography"], version = "^3.2.0"}
 passlib = {extras = ["bcrypt"], version = "^1.7.2"}
 python-multipart = "^0.0.5"
+psycopg2 = "^2.8.6"
```

Comment:
the lib psycopg2 is able to run in async flow? why the aoipg being removed?

Reply:
Yes, because gino has `asyncpg` as dependency
https://github.com/python-gino/gino/blob/master/pyproject.toml#L27

Reply:
Nice idea

Reply:
But I remember that had a problem with asyncpg and including I opened a issue [1] for this, but the problem seems to be in the pydantic enviroment. So I think that we can keep aiopg because of this problem and I think that the psycopg2 isn't necessary in this cenary. But if you want a try to test in new versions of pydantic and asyncpg, it looks interesting 

1 - https://github.com/tiangolo/fastapi/issues/1369

Reply:
Okay, I will add again the `aiopg = "^1.0.0"` about pydantic and asyncpg i will test in separated pull request.

Reply:
nice, but psycopg2 isn't necessary anymore

Reply:
ops.

Reply:
I tested locally and the alembic needs this dependency to run migrations,